src/borg/testsuite/helpers.py

```diff
@@ -405,6 +405,17 @@ def __repr__(self):
         return f"{self.id}: {self.ts.isoformat()}"
 
 
+# This is the local timezone of the system running the tests.
+# We need this e.g. to construct archive timestamps for the prune tests,
+# because borg prune operates in the local timezone (it first converts the
+# archive timestamp to the local timezone). So, if we want the y/m/d/h/m/s
+# values which prune uses to be exactly the ones we give [and NOT shift them
+# by tzoffset], we need to give the timestamps in the same local timezone.
+# Please note that the timestamps in a real borg archive or manifest are
+# stored in UTC timezone.
+local_tz = datetime.now().astimezone().tzinfo
```

Comment:
Is this the easiest way to get the local timezone?

Reply:
guess it could also be done via time.localtime(), .tm_gmtoff and timezone(timedelta(seconds=tm_gmtoff)), but that's rather more complicated.

Reply:
Searching found lots of things.  `datetime.now().astimezone().tzinfo` was one option, but some people prefer `datetime.utcnow().astimezone().tzinfo` for some reason.

Reply:
I'll replace it by `datetime.now(tz=timezone.utc).astimezone(tz=None).tzinfo`.

This avoids default argument values and also naive datetime objects.

Some time ago, I already refactored a lot of other places to use `datetime.now(tz=timezone.utc)`.